automationTest/src/test/java/com/owncloud/android/test/ui/testSuites/Config.java

```diff
@@ -37,18 +37,28 @@
 	public static final String userAccount = user + "@"+ URL;
 	public static final String userAccount2 = user2 + "@"+ URL2;
 
-	public static final String gmailAccount = "gmailAccountVar";
-	
-	public static final String fileWhichIsInTheServer1 ="test";
-	public static final String fileWhichIsInTheServer2 ="test";
-	
-	public static final String fileToTestName = "test";
-	public static final String fileToTestSendByEmailName = "test";
-	public static final String bigFileToTestName = "test";
-	
-	public static final String passcode1 = "passcode1";
-	public static final String passcode2 = "passcode2";
-	public static final String passcode3 = "passcode3";
-	public static final String passcode4 = "passcode4";
+	public static final String gmailAccount = "automationOwncloud@gmail.com";
 
+	public static final String fileWhichIsInTheServer1 ="ownCloudUserManual.pdf";
```

Comment:
We should refactor these to a properties file to avoid rebuilding after configuration.

For the future